new-dti-website/components/team/MemberDisplay.tsx

```diff
@@ -117,7 +117,7 @@ const MemberDisplay: React.FC = () => {
               >
                 {orderedAlumni.map((member, index) => (
                   <a
-                    href={member.linkedin ?? undefined}
+                    href={!member.linkedin ? undefined : member.linkedin}
```

Comment:
Previously this was just a nullish check, which handles the case when Linkedin profile field is "null" but not when it's an empty string.